### DISCUSSION TITLE
Help with meteorite relationships

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4638] [General]
We now have a whole bunch of meteorites in Arctos! Check out [UNM:MET](https://arctos.database.museum/SpecimenSearch.cfm?guid_prefix=UNM%3AMET)

There are some peculiar things related to this collection that I would like more people than me think about and perhaps offer up solutions for handling the data in Arctos.

Some of the catalog records are for things that were sampled from other catalog records. I could link them all together using "same individual as" relationships, but that does not provide the more nuanced information "this was prepared from that". For the objects that were the result of a sampling, I plan to add "creation" events, but that still doesn't let me say explicitly, this was created from that. An example of this relationship can be seen in these three records:

 "parent" - https://arctos.database.museum/guid/UNM:MET:13
 "child_1" - https://arctos.database.museum/guid/UNM:MET:15
 "child_2" - https://arctos.database.museum/guid/UNM:MET:17

Does "creation" make sense here? I feel like it does, because a person performed the sampling and prepared the sample, but maybe these should be "preparation" events?

I am thinking that it would also be good to have an event of some sort on the parent record that tells users it has been sampled. New kind of "collecting" (this really needs to change) event? If so what would it be?

And while the events make sense for recording the who/when/where, the actual relationships between the objects will need to be between their identifiers. I feel that "same individual as" seems wrong, and that maybe we need two new relationships sampled from/samples as?  I considered using parent/offspring, but that is technically not correct?

Other geological collections may have similar issues in the future, so I'd like to come up with a plan that others could use as well.

@beth3ha @Nicole-Ridgwell-NMMNHS @wellerjes @aklompma 

Also happy to have virtual coffee and discuss.....

**Comment by dustymc** [comment]
> I could link them all together using "same individual as" 

Seems correct to me. Maybe we need to talk about the terminology (or not?), but this seems like the same situation as splitting up skin and bones.

>does not provide the more nuanced information

Nor should it - it's a relationship between data objects representing concepts (eg "whatever someone felt like cataloging").

>"this was prepared from that".

"This" and "that" imply physical - parts. Hacking a thin section off of a meteorite isn't (much) different than cutting a subsample off of a frozen tissue. The sampling effects usability, parts are what's usable, that's best recorded as part attributes regardless of collection type.

>plan to add "creation" events,

That's probably "wrong" in that events are linked to virtual things rather than physical, but it's also (mostly, I think) consistent with what happens in cultural collections - I'm not entirely sure it's necessary, but its close enough to "normal" that I wouldn't object.

>parent/offspring, but that is technically not correct?

I don't think it's any other kind of correct either - there's no new material here, just preparations.

>plan that others could use as well.

Yep.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Is there some way we could expand upon the subsample relationship in parts by making subsample relationships possible between parts of different specimens?

**Reply by Jegelewicz** [reply]
I think that would be the best possible solution.

**Comment by dustymc** [comment]
>making subsample relationships possible between parts of different specimens?

In the model - sure. In the UI: uuuuhhhhhh, maybe, somehow..... That's definitely not "no" but I'd need a lot of guidance to write such a thing.

Very generally, a lot of this looks procedural - we spend a fair bit of time dealing with problems caused by cataloging things that didn't need cataloged, or not cataloging things that do, and of course Arctos is (AFAIK) the only thing that has any flexibility in that and we as a community are still figuring out what works best. My instinct is that needing a hard structural link between parts and across records is caused by cataloging the wrong things, but I wouldn't bet the farm on that theory either.

**Reply by Jegelewicz** [reply]
They are simply cataloging things as they get them.

1. get a meteorite, catalog it
2. take a slice of that meteorite (subsample the original meteorite), catalog it

This is a very logical way to do things when each of these pieces can then lead different lives. Sure, they could put all the "parts" in one record - but assigning specific attributes, preparators and identifiers to the individual parts has been problematic for other things (like what I spent time doing yesterday). Meteorite people are VERY interested in weight - understanding which weight attribute goes with which part is a good example of why cataloging them separately makes sense.

Anyway, because the meteorites are a one part to one catalog number relationship, I could relate them to each other through other identifiers, I just think "parent" and "child" don't feel right, although the natural alternative "parent sample" and "child sample" just seem redundant....

Sigh
